sips/sip-021/sip-021-nakamoto.md

```diff
@@ -0,0 +1,1051 @@
+to allocate block rewards to miners.  Unlike Stacks today, there is no 40/60 fee
+split between two consecutive miners.  Each miner nominally receives the entire
+coinbase and transaction fees before the MEV mitigation is applied.
+
+In the ATC-C algorithm, Nakamoto would use the document's recommended assumed
+total commitment function: the median total PoX spend across all miners for the
+past ten Bitcoin blocks.  It would additionally use the document's recommended
+carryforward function for missed sortitions' coinbases:  the coinbase for a
+Bitcoin block without a sortition would be available to winning miners across
+the next ten tenures.  That is, if a miner whose tenure begins during the next
+ten tenure-changes manages to produce a Stacks block with a `Coinbase`, then
+they receive a 20% of the coinbase that was lost.
+
+The reason ATC (and ATC-C) were not considered as viable anti-MEV strategies
+before is because a decrease in the PoX total spend can lead to a Bitcoin block
+with no sortition. This is a deliberate design choice in ATC-C, because it has
+the effect of lowering the expected return of MEV mining.  In ATC-C, the
+probability of a miner winning a sortition is _equal to_ (i.e. no longer
+proportional to) the miner's BTC spend, divided by the maximum of either the
+assumed total commit (median total BTC spend in the last 5 blocks) or the total
+BTC spend in this Bitcoin block.  This means that the sum of each miners'
+winning probabilities is not guaranteed to be 1.  The system deals with this by
+creating a virtual "nul" miner that participates in each sortition, such that
+its probability of the null miner winning is _1 -
+sum(probabilities-of-all-other-miners)_.  If the null miner wins, then the
+sortition is treated as empty.
+
+While the existence of a null miner was a liveness concern in Stacks today, it
+is not a concern in this proposal.  If the null miner wins tenure _N_, then the
+last non-null miner continues to produce blocks in tenure _N_.  They receive
+transaction fees, but no coinbase for tenure _N_.
+
+This proposal advocates for one additional change to ATC-C as described in the
+above report:  if a miner does not mine in at least five of the ten prior
+Bitcoin blocks, it has zero chance of winning.  This requires a Bitcoin MEV
+miner to participate as an honest miner for the majority of blocks it produces,
+such that even if they pay a _de minimis_ PoX payout each time, they are still
+paying Bitcoin transaction fees to other miners.
+
+### Example
+
+The need for this additional tweak becomes apparent when considering the
+consequences for a real Bitcoin MEV miner who is active on the Stacks network
+today: F2Pool.
+
+Consider what happens to F2Pool, who spends 200 sats on PoX and zero sats on
+transaction fees for their block-commit. Suppose the median total BTC spend over
+the last ten Bitcoin blocks was 500,000 sats (about what it is right now).  With
```

Comment:
semantics... but I would change "about what it is right now" to "about what it is at the time of writing" to be clear to the reader that this doc may have an out of date number